lib/runtime/wasmer/imports.go

```diff
@@ -895,10 +895,52 @@ func ext_trie_blake2_256_ordered_root_version_1(context unsafe.Pointer, dataSpan
 }
 
 //export ext_trie_blake2_256_ordered_root_version_2
-func ext_trie_blake2_256_ordered_root_version_2(context unsafe.Pointer,
-	dataSpan C.int64_t, version C.int32_t) C.int32_t {
-	// TODO: update to use state trie version 1 (#2418)
-	return ext_trie_blake2_256_ordered_root_version_1(context, dataSpan)
+func ext_trie_blake2_256_ordered_root_version_2(context unsafe.Pointer, dataSpan C.int64_t, version C.int32_t,
+) C.int32_t {
+	logger.Debug("executing...")
```

Comment:
Does this work? It shouldn't since v1 state trie is still not there. Why is it needed here as well?

Also this was implemented already in [#host functions](https://github.com/ChainSafe/gossamer/pull/2664)

Reply:
Looks like things work even without changing this function, removed it.